docs/content/en/docs/packages/cluster-autoscaler/addclauto.md

```diff
@@ -130,14 +125,20 @@ eksctl anywhere delete package --cluster <cluster-name> cluster-autoscaler
 
 ## Install Cluster Autoscaler in workload cluster
 
+  {{% alert title="Important" color="warning" %}}
+
+   While following this guide to install packages on a workload cluster, please make sure the `kubeconfig` is pointing to the management cluster that was used to create the workload cluster. The only exception is when you need to create a new namespace, the command `kubectl create <namespace>` should be executed with `kubeconfig` pointing to the workload cluster.
+   
+  {{% /alert %}}
+
 A few extra steps are required to install cluster autoscaler in a workload cluster instead of the management cluster.
 
 First, retrieve the management cluster's kubeconfig secret:
 ```yaml
 kubectl -n eksa-system get secrets <management-cluster-name>-kubeconfig -o yaml > mgmt-secret.yaml
 ```
 
-Update the secret's namespace to the namespace in the workload cluster that you would like to deploy the cluster autoscaler to.
+Update the secret's namespace to the `targetNamespace`, the namespace in the workload cluster that you would like to deploy the cluster autoscaler to.
```

Comment:
Would we want to recommend that kube-system be used as the targetNamespace? I was thinking that you wouldn't want to risk cluster autoscaler ever deploying on a node that it controlled.